docs/src/user_docs/assembly/instruction_reference.md

```diff
@@ -0,0 +1,325 @@
+# Miden VM Instruction Reference
+
+This page provides a comprehensive reference for Miden Assembly instructions.
+
+## Field Operations
+
+### Comparison Operations
+
+| Instruction                       | Stack Input | Stack Output | Cycles      | Notes                                                                      |
+| --------------------------------- | ----------- | ------------ | ----------- | -------------------------------------------------------------------------- |
+| `lte` <br> `lte.b`                | `[b, a, ...]` | `[c, ...]`   | 15 <br> 16   | `c <- 1` if `a <= b` <br> `c <- 0` otherwise                               |
```

Comment:
It'd be awesome if this could use the TeX rendering like in the "Instructions" tab on the playground, e.g.:

![image](https://github.com/user-attachments/assets/22d117a8-0e07-4762-896e-2ceb31229866)

Playground: https://0xmiden.github.io/examples/

Reply:
Thanks, looks awesome!